chirp/drivers/tdh8.py

```diff
@@ -710,7 +1006,10 @@ def get_features(self):
         return rf
 
     def process_mmap(self):
-        self._memobj = bitwise.parse(MEM_FORMAT, self._mmap)
+        if self.MODEL in H8_LIST:
+             self._memobj = bitwise.parse(MEM_FORMAT, self._mmap)
+        else:
+            self._memobj = bitwise.parse(MEM_FORMAT_H3, self._mmap)
```

Comment:
This would be better as a subclass override, much like `_get_nam()` and some others.